src/book.py

```diff
@@ -834,6 +834,37 @@ def detect_exchange(self, file_path: Path) -> Optional[str]:
 
         return None
 
+    def get_price_from_csv(self):
+        for platform, operations_a in misc.group_by(
+            self.operations, "platform"
+        ).items():
+            for timestamp, operations_b in misc.group_by(
+                operations_a, "utc_time"
+            ).items():
+                if len(operations_b) > 1:
+                    buytr = selltr = None
+                    buycount = sellcount = 0
+                    for operation in operations_b:
+                        if isinstance(operation, tr.Buy):
+                            buytr = operation
+                            buycount += 1
+                        elif isinstance(operation, tr.Sell):
+                            selltr = operation
+                            sellcount += 1
+
+                    if buycount == 1 and sellcount == 1:
+                        price = decimal.Decimal(buytr.change / selltr.change)
```

Comment:
Shouldn't this be sell/buy?

Reply:
Yes but it works and i dunno why if we use sell/buy all prices are inverted in the DB (eur/btc=30k) and if we use buy/sell there Not inverted (eur/btc=0.00XXX) and i havent found out why yet

Reply:
I'm also confused now.
Could it be that the prices are actually inverted in the database? If one BTC costs 30k EUR, then BTC/EUR should be 1/30k, right? But the database stores it the other way around.

Reply:
If I use `price = decimal.Decimal(selltr.change / buytr.change)` and replace `if reciprocal` by `if not reciprocal` in `price_data.py` (line 661), then I get correct calculations for both the CSV prices and the API prices (tested with Kraken).

Reply:
> I'm also confused now. Could it be that the prices are actually inverted in the database? If one BTC costs 30k EUR, then BTC/EUR should be 1/30k, right? But the database stores it the other way around.

Btc/eur should be 30k if btc costs 30k

Reply:
I see two possible ways of defining the price:
1. The "physical" price definition: Trade 30k EUR for 1 BTC:
BTC/EUR = 1 (BTC) / 30k (EUR) = 1/30k
In this case, the price should be `price = decimal.Decimal(selltr.change / buytr.change)`.
2. The price definition used for charts on various exchanges:
BTC/EUR = price of one BTC in EUR = 30k
In this case `buytr.change / selltr.change` is already correct. But this somehow messes with data that is fetched via the APIs.

Reply:
I'd Stick with 2. Because thats what we get from All APIs.

Reply:
Alright, then this calculation should be right.